render.yaml

```
@@ -0,0 +1,94 @@
+databases:
+- name: temporal
+  databaseName: temporal
+  user: temporal
+  plan: standard
+- name: temporal-visibility
+  databaseName: visibility
+  user: visibility
+  plan: standard
+
+services:
+- type: pserv
```

```suggestion
- type: web
```

this service needs to be exposed to the internet so that SDKs can connect to it. is it just a oneliner change?

---

It is